tests/imposter/private-api-config.yaml

```diff
@@ -133,6 +133,28 @@ resources:
           "redirect_uri": "http://example.net"
         }
 
+  - method: GET
+    path: /question
+    requestHeaders:
+      session-id: "authorization-error"
+    response:
+      statusCode: 202
```

Comment:
What is `202 Accepted` being used for? 

https://github.com/govuk-one-login/ipv-cri-tech-docs/blob/main/doc/diagrams/kbv-hmrc-api-flow.md describes the interactions between the FE and the API, which only lists:
- 200 OK
- 204 No Content